app/resources/api/v2/request_metadata_resource.rb

```diff
@@ -43,6 +43,13 @@ class RequestMetadataResource < BaseResource
       #     pooling.
       attribute :cells_per_chip_well, write_once: true
 
+      # @!attribute [r] cells_per_chip_well
+      #   @return [String] the allowance_band requested in the Submission. As
+      #     used in the scRNA Core pipeline, it is specified at the Study-Project
+      #     level: it will have the same value for all Requests that share the
+      #     same Study and Project.
+      attribute :allowance_band, write_once: true
```

Comment:
I guess this does not currently have to be writeable over the API, as it's calculated internally - so perhaps it should be read only?